include/envoy/http/header_map.h

```diff
@@ -308,7 +308,10 @@ class HeaderEntry {
   HEADER_FUNC(EnvoyUpstreamServiceTime)                                                            \
   HEADER_FUNC(Location)                                                                            \
   HEADER_FUNC(Server)                                                                              \
-  HEADER_FUNC(Status)
+  HEADER_FUNC(Status)                                                                              \
+  HEADER_FUNC(XRateLimitLimit)                                                                     \
+  HEADER_FUNC(XRateLimitRemaining)                                                                 \
+  HEADER_FUNC(XRateLimitReset)
```

Comment:
This should be defined as custom inline headers for the ratelimit header vs. global inline headers. See `RegisterCustomInlineHeader`